.babelrc

```diff
@@ -15,5 +15,6 @@
         "transform-react-remove-prop-types"
       ]
     }
-  }
+  },
+  "sourceMaps": true
```

Comment:
This setting is ignored when used with webpack. See **Note** at https://github.com/babel/babel-loader#user-content-options

Reply:
Good catch.